src/components/MusicPlayer/ThePlayerBar.vue

```diff
@@ -393,6 +393,20 @@ It can be an album/playlist or the original song lists where this song is from
 
   created() {
     window.addEventListener('keydown', this.handleKeyDown);
+
+    //Add MediaKey Support using MediaSession
+    navigator.mediaSession.setActionHandler("nexttrack", _ => {
```

Comment:
Should check if mediaSession is available on navigator before using it

`if ('mediaSession' in navigator)`

More on this: https://developer.mozilla.org/en-US/docs/Web/API/Media_Session_API